test/unit/support.js

```diff
@@ -67,7 +67,27 @@ testIframeWithCallback( "Check CSP (https://developer.mozilla.org/en-US/docs/Sec
 			"pixelMarginRight": true,
 			"pixelPosition": true,
 			"radioValue": true,
-			"reliableMarginRight": true
+			"reliableMarginRight": true,
+			"reliableDimensionsInFullscreenMode": true
+		};
+	} else if ( /rv:11\.0/i.test( userAgent ) ) {
+		expected = {
+			"ajax": true,
+			"boxSizingReliable": true,
```

Comment:
This should be `false`. Generally, you can copy over the shared test results of IE10/11 and just add your test, the only one that will differ.